### DISCUSSION TITLE
C++ Qt Integration

### DISCUSSION THREAD

**Discussion by zethon** [microsoft/monaco-editor#4192] [Q&A]
I see that there is a PyQt integration here: https://pypi.org/project/monaco-qt/

Is there any way to use Monaco in a Qt C++ application?

**Comment by 0xdevalias** [comment]
While I haven't looked too deeply into it, based on the project you linked above, you could probably do the equivalent in Qt C++; there didn't seem to be a whole lot of 'glue code'.

- https://pypi.org/project/monaco-qt/
  - https://github.com/DaelonSuzuka/monaco-qt
    - > The `monaco` text editor deployed as a Qt Widget.
      > Uses `QWebEngineView` to load a custom web page containing an instance of `monaco`.
      - https://doc.qt.io/qt-6/qwebengineview.html
        - > QWebEngineView Class
          > The QWebEngineView class provides a widget that is used to view and edit web documents.
      - https://doc.qt.io/qt-6/qtwidgets-index.html
        - > Qt Widgets
